backoffice/management/tests/test_migrations.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+
+@pytest.mark.django_db
+def test_add_groups(migrator):
+    migrator = migrator()
+    migrator.migrate("management", "0001_initial")
+    Group = migrator.apps.get_model("auth", "Group")
+    all_groups = Group.objects.all()
+    assert len(all_groups) == 2
```

Comment:
TBH I wouldn't test the migration, we can trust django that does work. Just the integration tests of the actually groups should be sufficient.